Library/RewardStateTests.swift

```diff
@@ -168,4 +168,59 @@ class RewardStateTests: TestCase {
       RewardCellProjectBackingState.state(with: project, reward: reward)
     )
   }
+
+  func test_noRewardType_BackedNonLive() {
+    let reward = Reward.noReward
+      |> Reward.lens.remaining .~ 0
```

Comment:
I think we shouldn't modify these properties on the `.noReward` template, because the template already mimics what gets returned from the server 🤔.